DataFormats/Math/interface/constexpr_cmath.h

```diff
@@ -0,0 +1,10 @@
+#ifndef DataFormats_Math_constexpr_cmath_h
+#define DataFormats_Math_constexpr_cmath_h
+
+namespace reco {
+  constexpr int32_t ceil(float num) {
```

Comment:
the header consistency check is complaining about int32_t ; which means this file needs an include of ```<cstdint>```

Reply:
done